### PR TITLE
Add the CREATE USER and DROP USER commands

### LINKED ISSUE #145: Document "create user" and the default super user
The related PR is https://github.com/singularity-data/risingwave/pull/3074

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
- **Description**: 
Add the CREATE USER and DROP USER commands

- **Related code PR**: 
https://github.com/singularity-data/risingwave/pull/3074

- **Related doc issue**: 
Resolves #145 

- **Notes**: 
Further changes are expected in v0.1.12 or future versions. 
Track: https://github.com/singularity-data/risingwave/pull/4414 ; https://github.com/singularity-data/risingwave/issues/4428; https://github.com/singularity-data/risingwave/issues/4440

## Before merging
  - [x] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked related doc issue to this PR in **Development**.
  - [ ] I have accquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [x] I have checked the doc site preview and the updated parts look all good. <details><summary>How?</summary>image</details>
  - [x] (For version-specific PR) I have confirmed that this PR is for a released version. If the software version is not released, put it on hold.

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by yezizp2012 — APPROVED** [review]

File: docs/sql/commands/sql-create-user.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Parameters
+| Parameter or clause | Description           |
+| ------------------- | --------------------- |
+| *user_name* | The name of the user account to be created. |
+| **WITH** *option* | See the table below. |
+
+#### `WITH` options
+| Parameter | Description           |
+| --------- | --------------------- |
+| SUPERUSER <br/><br/> NOSUPERUSER | Grants/denies the user superuser permission. A superuser can override all access restrictions. <br/> If not specified, NOSUPERUSER is the default. |
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-user.md]: Would the information be presented better if we use unordered list (that is, `<ul><li></li></ul>`) for the possible values of a parameter?

Comment (hengm3467) [inline docs/sql/commands/sql-create-user.md]: We can rephrase this sentence to be either of these:
- If not specified, NOSUPERUSER will be used.
- NOSUPERUSER is the default value.

File: docs/sql/commands/sql-create-user.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| CREATEDB <br/><br/> NOCREATEDB | Grants/denies the user the ability to create databases. <br/> If not specified, NOCREATEDB is the default. |
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-user.md]: Same suggestion as in the comment above.

Comment (hengm3467) [reply docs/sql/commands/sql-create-user.md]: The comment also applies to `LOGIN`.

File: docs/sql/commands/sql-create-user.md (hunk 1 above)
Comment (CharlieSYH) [reply docs/sql/commands/sql-create-user.md]: NOSUPERUSER is the default value.
NOCREATEDB is the default value.
LOGIN is the default value.

Comment (CharlieSYH) [reply docs/sql/commands/sql-create-user.md]: | Parameter | Description           |
| --------- | --------------------- |
| <ul><li>SUPERUSER</li><li>NOSUPERUSER</li></ul> | Grants/denies the user superuser permission. A superuser can override all access restrictions. <br/> NOSUPERUSER is the default value. |

Comment (hengm3467) [reply docs/sql/commands/sql-create-user.md]: Looks good!

**Review by hengm3467 — APPROVED** [review]
LGTM

**Pull request merged**